packages/core/test/buttons/buttonTests.tsx

```diff
@@ -47,6 +47,11 @@ function buttonTestSuite(component: React.ComponentClass<any>, tagName: string)
             assert.equal(wrapper.text(), "some text");
         });
 
+        it("renders the button text prop", () => {
+            const wrapper = mount(<Button data-test-foo="bar" />);
+            assert.isTrue(!!wrapper.find('[data-test-foo="bar"]'));
```

Comment:
please avoid implicit type coercion to boolean. instead:

```suggestion
            assert.isTrue(wrapper.find('[data-test-foo="bar"]').exists());
```